pycparser/c_lexer.py

```diff
@@ -209,6 +209,10 @@ def _make_tok_location(self, token):
 
     bad_octal_constant = '0[0-7]*[89]'
 
+    # comments are not supported
+    unsupported_c89_comment = r'\/\/'
```

Comment:
I don't understand the c89/c99 distinction here. Aren't `//` just C++ style comments and `/*` the original C style?

Reply:
I thought it made more sense to describe the comments from a C perspective, rather than a C++ perspective.  I'll update this.

Also, I accidentally swapped c89 / c99.